ush/templates/FV3LAM_wflow.xml

```diff
@@ -263,7 +283,7 @@ MODULES_RUN_TASK_FP script.
     <dependency>
       <!-- Files through fhr 39 exist. -->
       <sh>
-        test $(find <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}/@Y@m@d@H/postprd</cyclestr> \
+        test $(find <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}/@Y@m@d@H/postprd/</cyclestr> \
                -name wrfnat_rr_[0-3][0-9].grib2 -mmin +5 | wc -l) \
         == 40
```

Comment:
This dependency is only satisfied if the RAP forecast length is 39 hours (0 thru 39 = 40 files), or if you just happen to catch it when wrfnat_rr_xx.grib2 files have been generated for 40 of the 52 hours.  This will probably lead to a lot of missed dacycles.  I think you need some logic in here to look for 52 files when processing 03,09,15,21z RAP, or 22 files at all other hours

Reply:
Trevor,

I found this issue and manually changed this check to find smaller number of the RAP forecast files in dev2.
I will leave this problem to the future PR. We need to check the forecast length for boundary. We also should let this step start if certain time passed and the scripts should check older RAP cycles for the boundary. 

Thanks,
Ming